application/backend/eye_tracker_newsdk.py

```diff
@@ -18,8 +18,8 @@
 from tornado import gen
 import emdat_utils
 import ast
-from websocket_client import EyetrackerWebsocketClient
 import subprocess
+from application.backend.eye_tracker_class import EyeTracker, SimulationEyeTracker, Tobii4CEyeTracker, TobiiT120EyeTracker
```

Comment:
just checking if we can just import all (*)? probably easier if later on we add more eye-trackers